COMPILING.md

```diff
@@ -0,0 +1,110 @@
+# Compiling OpenTTD
+
+## Windows:
+
+You need Microsoft Visual Studio 2015 Update 3 or more recent. Open the project file
+and it should build automatically. In case you want to build with SDL support
+you need to add WITH_SDL to the project settings.
+
+PNG (WITH_PNG), ZLIB (WITH_ZLIB), LZO (WITH_LZO), Freetype (WITH_FREETYPE) and
+libLZMA (WITH_LIBLZMA) support is enabled by default. For these to work you need their
+development files. To get them just use vcpkg from https://github.com/Microsoft/vcpkg
+using x86-windows-static and x64-windows-static triplets.
+For more help with VS see docs/Readme_Windows_MSVC.md.
+
+You can also build it using the Makefile with MSYS/MinGW or Cygwin/MinGW.
+Please read the Makefile for more information.
+
+
+The OpenTTD wiki may provide additonal help with [compiling for Windows](https://wiki.openttd.org/Category:Compiling_OpenTTD).
+
+## Solaris, FreeBSD, OpenBSD:
```

Comment:
No need for a separate section, IMO. Just merge with Linux/Unix (BSDs are Unixes, after all)

Reply:
Done